data/ny/retired/Assembly-District--a245f8a1-a310-4b5e-baaa-1ec2c060721e.yml

```diff
@@ -0,0 +1,22 @@
+id: ocd-person/a245f8a1-a310-4b5e-baaa-1ec2c060721e
+name: Assembly District 49
+image: https://assembly.state.ny.us/write/upload/member_files/000/headshot/000.jpg
```

Comment:
@bfossen-ce same thing here. This district is held by Lester Chang - https://ballotpedia.org/Lester_Chang.

Reply:
At the time of the scrape this seat was still in a recount. Will delete this record